packages/env/src/web.ts

```diff
@@ -27,7 +27,7 @@ export const env = createEnv({
   },
   client: {},
   experimental__runtimeEnv: {
-    NODE_ENV: process.env['NODE_ENV'],
+    NODE_ENV: process.env.NODE_ENV,
   },
   emptyStringAsUndefined: true,
   skipValidation: !!process.env['SKIP_ENV_VALIDATION'],
```

Comment:
Curious why eslint doesn't have a problem with ['SKIP_ENV_VALIDATION']

Reply:
because `NODE_ENV` is known to be on `process.env` but `SKIP_ENV_VALIDATION` isn't